packages/@dcl/inspector/package.json

```diff
@@ -30,6 +27,7 @@
     "react-dnd": "^16.0.1",
     "react-dnd-html5-backend": "^16.0.1",
     "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
```

Comment:
is it necessary to add a new dep for file dropping? we already have `react-dnd` which supposedly support that 

https://react-dnd.github.io/react-dnd/examples/other/native-files